diff --git a/node_modules/tldraw/dist-esm/lib/ui/components/primitives/TldrawUiTooltip.mjs b/node_modules/tldraw/dist-esm/lib/ui/components/primitives/TldrawUiTooltip.mjs
index c5e9a5d..aeac108 100644
--- a/node_modules/tldraw/dist-esm/lib/ui/components/primitives/TldrawUiTooltip.mjs
+++ b/node_modules/tldraw/dist-esm/lib/ui/components/primitives/TldrawUiTooltip.mjs
@@ -174,8 +174,11 @@ function TooltipSingleton() {
       const activeRect = currentTooltip.targetElement.getBoundingClientRect();
       const trigger = triggerRef.current;
       trigger.style.position = "fixed";
-      trigger.style.left = `${activeRect.left}px`;
-      trigger.style.top = `${activeRect.top}px`;
+      trigger.style.left = "0px";
+      trigger.style.top = "0px";
+      const cbOffset = trigger.getBoundingClientRect();
+      trigger.style.left = `${activeRect.left - cbOffset.left}px`;
+      trigger.style.top = `${activeRect.top - cbOffset.top}px`;
       trigger.style.width = `${activeRect.width}px`;
       trigger.style.height = `${activeRect.height}px`;
       trigger.style.pointerEvents = "none";
